lib/inject.rb

```diff
@@ -0,0 +1,10 @@
+class Array
+
+	def inject_method(array)
+		each do |element|
+			array = yield array, element
+		end
+		p array
```

Comment:
You shouldn't need to `p` the results, as you want the method simply to return the array